src/linux-distro.ts

```diff
@@ -104,10 +104,22 @@ function listDistroIds({ id, version, codename }: { id: string, version: string,
       return [{ value: 'rhel' + version + '0', priority: 100 }];
     case 'redhatenterprise':
     case 'redhatenterpriseserver': {
+      const toRhelVersions = (v: number, i: number) => {
+        if (v === 80) {
+          return [
+            { value: 'rhel80', priority: (i + 1) * 100 },
+            { value: 'rhel8', priority: ((i + 1) * 100) + 1 }
+          ];
+        } else {
+          return [{ value: 'rhel' + v, priority: (i + 1) * 100 }];
+        }
+      };
+
       const want = +version.replace('.', '');
       const known = [55, 57, 62, 67, 70, 71, 72, 80, 81, 82, 83, 90];
       const allowedVersions = known.filter(v => v <= want);
-      return allowedVersions.map((v, i) => ({ value: 'rhel' + v, priority: (i + 1) * 100 }));
+      const priorities = allowedVersions.map(toRhelVersions);
+      return flattenArray(priorities);
```

Comment:
Based on the test matrix, the minimum supported node version is 12 and it should have flatmap

```suggestion
      return allowedVersions.flatMap(toRhelVersions);
```

Reply:
Then I'll need to upgrade the tsconfig.json to es2019 so it compiles. If I keep it as es2018 we don't have flatmap.

Reply:
Ah, typescript config is in the way, got it! I'm pretty sure Node.js 12 is fully es2019 compatible, but if bumping the version causing any other unexpected issues, feel free to just resolve this suggestion 🙂